### DISCUSSION TITLE
Is OVPN wireguard working with gluetun?

### DISCUSSION THREAD

**Discussion by allelouia** [qdm12/gluetun#1743] [Q&A]
Hi,
I'm talking about the VPN provider, not openvpn config file :)
I found few discussions about problems with OVPN but I'm not sure if they are solved or not.
So are there some working wireguard ovpn connections with gluetun here?
Thank you

**Comment by gmillerd** [comment]
What happens when you use the custom provider? What is different about OVPN?

https://www.ovpn.com/en/faq/wireguard

**Reply by allelouia** [reply]
I don't use OVPN actually but I've got to change since IVPN will stop port forwarding.
I'm inquiring because I've seen several threads (one or two years old) mentioning problems with OVPN.
